astropy/coordinates/baseframe.py

```diff
@@ -1278,8 +1279,10 @@ def _frameattr_equiv(left_fattr, right_fattr):
                 return True
             elif isinstance(right_fattr, r.BaseRepresentationOrDifferential):
                 # both are representations.
-                if ((hasattr(left_fattr, 'differentials') and left_fattr.differentials) or
-                    hasattr(right_fattr, 'differentials') and right_fattr.differentials):
+                if ((hasattr(left_fattr, 'differentials') and
```

Comment:
It is actually less readable broken into 4 lines like this, at least for me.

Reply:
I think this should just be
```
if (getattr(left_fattr, 'differentials', False) or
        getattr(right_fattr, 'differentials', False)):
```

Reply:
Agree with @mhvk!